.evergreen/prep_c_driver_source.sh

```diff
@@ -3,12 +3,8 @@
 set -o xtrace
 set -o errexit
 
-MONOGO_C_DRIVER_REF=1.17.0
-
-# Force checkout with lf endings since .sh must have lf, not crlf on Windows
-git clone git@github.com:mongodb/mongo-c-driver.git --config core.eol=lf --config core.autocrlf=false
+./libmongocrypt/.evergreen/clone-mongo-c-driver.sh
```

Comment:
Should this also use `$libmongocrypt_root`?

```suggestion
$libmongocrypt_root/.evergreen/clone-mongo-c-driver.sh
```

Reply:
Done.